proposed/http-client/http-client-meta.md

```diff
@@ -82,6 +82,21 @@ and second by catching potential exceptions.
 To make the specification more predictable, it was decided that HTTP clients never will throw
 exceptions for 4xx and 5xx responses.
 
+#### Why "sendRequest"?
+
+The method signature `ClientInterface::sendRequest(RequestInterface $request): ResponseInterface`
+might sound a bit verbose at first. An obvious improvement would be to just
+call the method `send`.
+
+The reason behind this verbosity is how existing HTTP client libraries are implemented.
+Most of them already have a method called `send` with an arbitrary signature.
+To make adoption easier we decided to choose a more verbose name.
+Besides the backward compatibility argument, future HTTP client implementations
```

Comment:
i would drop this second part. if we called our thing `send`, the future clients could call their convenience method `get` and `post` etc, for example.